docs/home/defend_your_workstation.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Defend Your Workstation
+category: 6255e67693d5200013b1fa40
+hidden: false
+---
+
+[Developers are the new high-value targets](https://blog.phylum.io/developers-are-the-new-high-value-targets/). Phylum can defend developers where they work!
+
+In addition to [integrations offered](https://docs.phylum.io/docs/integrations_overview) to protect applications and CI pipelines, Phylum offers a solution to defend developer workstations.
+
+# Overview
+
+Phylum uses a two-pronged approach to protect your system during package installation: package pre-check and an installation sandbox.
+
+## How It Works
+
+The Phylum CLI offers default ecosystem extensions that wrap package manager tools to protect commands which would install new packages or otherwise execute arbitrary code.
+
+When using Phylum to install a package with an ecosystem extension, the application will first query the Phylum platform for any findings related to the packages that would be installed. If the package findings fail the default policy, the installation is aborted.
+
+An additional layer of protection is provided with an installation sandbox. This ensures restricted resources like filesystem and network operations are enforced while executing arbitrary code. If any unwanted behavior is observed, Phylum will block the behavior and abort the installation before your system is impacted.
```

Comment:
The wording here is a bit vague on _when_ the sandbox is applied. Some readers may think it implies the sandbox is applied for _every_ ecosystem extension. Perhaps the wording can be better to call out that _some_ of the ecosystem extensions are protected with sandbox out of the box (as it were). These are currently the `npm`, `yarn`, and `pip` extensions.

Reply:
I added a note here

https://github.com/phylum-dev/documentation/blob/e807cfcd89cfcd794ce491b35c0997dc47b9fddc/docs/home/defend_your_workstation.md?plain=1#L101